src/features/client/pages/Subscriptions/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import RestaurantPromo1 from '../../components/RestaurantSignUp/RestaurantPromo1';
+import RestaurantPromo2 from '../../components/RestaurantSignUp/RestaurantPromo2';
+import SubscriptionPlans from '../../components/RestaurantSignUp/SubscriptionPlans';
+import SubmitForm from '../../components/RestaurantSignUp/SubmitForm';
+import { ChakraProvider } from '@chakra-ui/react';
+import './index.scss';
+
+function Subscriptions() {
+  return (
+    <ChakraProvider>
+      <RestaurantPromo1 />
```

Comment:
Dont use numbers in components names. Try naming it differently

Reply:
fixed